Src/Support/Google.Apis.Auth/OAuth2/ComputeCredential.cs

```diff
@@ -68,10 +68,12 @@ public class ComputeCredential : ServiceCredential, IOidcTokenProvider, IGoogleC
         public string OidcTokenUrl { get; }
 
         /// <inheritdoc/>
-        bool IGoogleCredential.HasExplicitScopes => false;
+        bool IGoogleCredential.HasExplicitScopes => HasExplicitScopes;
 
         /// <inheritdoc/>
-        bool IGoogleCredential.SupportsExplicitScopes => false;
+        bool IGoogleCredential.SupportsExplicitScopes => true;
```

Comment:
Could this break code? I'm imagining a case of:

- Using our client library with default credentials
- Running on GCE
- Using a *bad* set of scopes

... it currently works due to the scopes not being applied, but scopes will start being applied, and then fail.

Maybe that's okay?

Reply:
Okay, SGTM.